LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2022 Toni Rico <antonio.rico.diez@revenuecat.com>
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```

Comment:
What we use is:

//
//  Copyright RevenueCat Inc. All Rights Reserved.
//
//  Licensed under the MIT License (the "License");
//  you may not use this file except in compliance with the License.
//  You may obtain a copy of the License at
//
//      https://opensource.org/licenses/MIT
//
//  <filename>
//
//  Created by <Full name if you want> on <mm/dd/yy>
//

Reply:
Hmm that seems like the header we add to files in iOS. This is a `LICENSE` file which I believe is supposed to be more generic, explaining the license itself... I checked and we have the same `LICENSE` on the iOS and Android SDKs... That said, I'm not fully familiar with this, so happy to talk about it if you think we should change it!

Reply:
Sorry, I didn't see that, I thought it was a header in a file! You're right.
We still need to change it and attribute it back to RC 😄
`Copyright (c) 2022 Toni Rico <antonio.rico.diez@revenuecat.com>` -> `Copyright RevenueCat Inc. All Rights Reserved.`